packages/esm-cervical-cancer-app/webpack.config.js

```diff
@@ -11,7 +11,7 @@ config.overrides.resolve = {
   alias: {
     '@openmrs/esm-framework': '@openmrs/esm-framework/src/internal',
     '@ohri/openmrs-esm-ohri-commons-lib': path.resolve(__dirname, '../esm-commons-lib/src/index'),
-    '@ohri/openmrs-ohri-form-engine-lib': '@ohri/openmrs-ohri-form-engine-lib/src/index',
+    '@openmrs/openmrs-form-engine-lib': '@openmrs/openmrs-form-engine-lib/src/index',
```

Comment:
Do we still need this? do u remember why we added it?

Reply:
Yes, we do need the alias. Without it, the imports won't resolve. It relates to the library is setup